lib/ansible/utils/module_docs_fragments/netapp.py

```diff
@@ -38,8 +38,7 @@ class ModuleDocFragment(object):
   username:
       required: true
       description:
-      - Username: This can be a Cluster-scoped or SVM-scoped account, depending on whether a Cluster-level or SVM-level API is required. For more information, please read the documentation U(https://goo.gl/BRu78Z).
-
+      - This can be a Cluster-scoped or SVM-scoped account, depending on whether a Cluster-level or SVM-level API is required. For more information, please read the documentation U(https://goo.gl/BRu78Z).
```

Comment:
I see you have this change in a number of PRs. Once the first PR gets merged and you rebase the remaining PRs this change should disappear from the remaining PRs.